### PR TITLE
Give lists in tables uniform font size

### PR DESCRIPTION
**Pull request by jonalport**
Requirement is to keep font size of lists the same as the rest of the table

### CODE REVIEW

**Comment by Guntrisoft** [comment]
Obviously this is pretty generic, so I'm just wondering whether this will have any bad knock on effects elsewhere...!

**Comment by Tr4pSt3R** [comment]
This styling is pretty broad.. I can't imagine it not affecting some part of the site

**Comment by jonalport** [comment]
Fair point, @Guntrisoft & @Tr4pSt3R. Have name-spaced the style within editorial content.

File: app/assets/stylesheets/components/common/_lists.scss [hunk 1]
```diff
@@ -95,6 +95,15 @@
   @extend .list;
 }
 
+// Dough sets th, td to 14px. We want contained lists to match within editorial content.
+.editorial {
+  th, td {
+    ul, ol {
+      font-size: 14px;
```
Comment (Guntrisoft) [inline app/assets/stylesheets/components/common/_lists.scss]: Does using @include body() benefit here?

**Comment by Guntrisoft** [comment]
Can you rebase the branch so there's just one commit (squash)?

File: app/assets/stylesheets/components/common/_lists.scss (hunk 1 above)
Comment (jonalport) [inline app/assets/stylesheets/components/common/_lists.scss]: I'm just matching the line in basic.scss, so not sure I want to add anything extra.

**Comment by jonalport** [comment]
All cleaned up

**Comment by Guntrisoft** [comment]
:+1:

**Comment by Tr4pSt3R** [comment]
:+1:

**Comment by jonalport** [comment]
Tough crowd! Thanks :)

**Pull request merged**